pytorch/buildspec.yml

```diff
@@ -152,33 +152,33 @@ images:
     target: e3
     context:
       <<: *INFERENCE_CONTEXT
-  # BuildSageMakerCPUPTInferencePy3DockerImage:
-  #   <<: *INFERENCE_REPOSITORY
-  #   build: &PYTORCH_CPU_INFERENCE_PY3 false
-  #   image_size_baseline: 4899
-  #   base_image_name: BuildE3CPUPTInferencePy3DockerImage
-  #   device_type: &DEVICE_TYPE cpu
-  #   python_version: &DOCKER_PYTHON_VERSION py3
-  #   tag_python_version: &TAG_PYTHON_VERSION py38
-  #   os_version: &OS_VERSION ubuntu20.04
-  #   tag: !join [ *VERSION, "-", *DEVICE_TYPE, "-", *TAG_PYTHON_VERSION, "-", *OS_VERSION, "-sagemaker" ]
-  #   docker_file: !join [ docker/, *SHORT_VERSION, /, *DOCKER_PYTHON_VERSION, /Dockerfile., *DEVICE_TYPE ]
-  #   target: sagemaker
-  #   context:
-  #     <<: *INFERENCE_CONTEXT
-  # BuildSageMakerGPUPTInferencePy3DockerImage:
-  #   <<: *INFERENCE_REPOSITORY
-  #   build: &PYTORCH_GPU_INFERENCE_PY3 false
-  #   image_size_baseline: 14000
-  #   base_image_name: BuildE3GPUPTInferencePy3DockerImage
-  #   device_type: &DEVICE_TYPE gpu
-  #   python_version: &DOCKER_PYTHON_VERSION py3
-  #   tag_python_version: &TAG_PYTHON_VERSION py38
-  #   cuda_version: &CUDA_VERSION cu115
-  #   os_version: &OS_VERSION ubuntu20.04
-  #   tag: !join [ *VERSION, "-", *DEVICE_TYPE, "-", *TAG_PYTHON_VERSION, "-", *CUDA_VERSION, "-", *OS_VERSION, "-sagemaker" ]
-  #   docker_file: !join [ docker/, *SHORT_VERSION, /, *DOCKER_PYTHON_VERSION, /, *CUDA_VERSION, /Dockerfile.,
-  #                        *DEVICE_TYPE ]
-  #   target: sagemaker
-  #   context:
-  #     <<: *INFERENCE_CONTEXT
\ No newline at end of file
+  BuildSageMakerCPUPTInferencePy3DockerImage:
+    <<: *INFERENCE_REPOSITORY
+    build: &PYTORCH_CPU_INFERENCE_PY3 false
+    image_size_baseline: 4899
+    base_image_name: BuildE3CPUPTInferencePy3DockerImage
+    device_type: &DEVICE_TYPE cpu
+    python_version: &DOCKER_PYTHON_VERSION py3
+    tag_python_version: &TAG_PYTHON_VERSION py38
+    os_version: &OS_VERSION ubuntu20.04
+    tag: !join [ *VERSION, "-", *DEVICE_TYPE, "-", *TAG_PYTHON_VERSION, "-", *OS_VERSION, "-sagemaker" ]
+    docker_file: !join [ docker/, *SHORT_VERSION, /, *DOCKER_PYTHON_VERSION, /Dockerfile., *DEVICE_TYPE ]
+    target: sagemaker
+    context:
+      <<: *INFERENCE_CONTEXT
+  BuildSageMakerGPUPTInferencePy3DockerImage:
+    <<: *INFERENCE_REPOSITORY
+    build: &PYTORCH_GPU_INFERENCE_PY3 false
+    image_size_baseline: 14000
+    base_image_name: BuildE3GPUPTInferencePy3DockerImage
+    device_type: &DEVICE_TYPE gpu
+    python_version: &DOCKER_PYTHON_VERSION py3
+    tag_python_version: &TAG_PYTHON_VERSION py38
+    cuda_version: &CUDA_VERSION cu115
```

Comment:
I remember that e3 GPU images were released with cu115 and the SM GPU images were released with cu113, can you confirm please and make the changes if required?

Reply:
Yes, the sagemaker ones are released with cu113. Making the change and pushing a commit.